julia/deps/build.jl

```diff
@@ -54,7 +54,7 @@ if Sys.isunix()
   nvcc_path = Sys.which("nvcc")
   if nvcc_path ≢ nothing
     @info "Found nvcc: $nvcc_path"
-    push!(CUDAPATHS, replace(nvcc_path, "bin/nvcc", "lib64"))
+    push!(CUDAPATHS, replace(nvcc_path, "bin/nvcc" => "lib64"))
```

Comment:
ah, I have a PR that fixed the building issues https://github.com/apache/incubator-mxnet/pull/16125

Reply:
Great! Then this can PR can be ignored.

Reply:
:thinking: ... we still can get this merged into master first.